composer.json

```diff
@@ -96,11 +96,13 @@
         "roundcube/plugin-installer": "*"
     },
     "require": {
-        "composer-plugin-api": "^1.0"
+        "composer-plugin-api": "^1.0|^2.0"
```

Comment:
Please prefer `^1.0 || ^2.0` for clarity and as that's the recommended way. Same below at line 102.